PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
```

Comment:
I'm not sure it's a good idea to split `Description` and `Motivation / Context`.
What's the expected gain of this split over a unique `Description` section?